Code/Framework/AzCore/Platform/Android/AzCore/std/parallel/internal/thread_Android.cpp

```diff
@@ -25,11 +25,9 @@ namespace AZStd
             // Unimplemented on Android
         }
 
-        void SetThreadPriority(int priority, pthread_attr_t&)
+        void SetThreadPriority([[maybe_unused]] int priority, pthread_attr_t&)
```

Comment:
really small nit: (for consistency with the other functions here)
```suggestion
        void SetThreadPriority(int, pthread_attr_t&)
```

Reply:
Not worth the cost of another AR :) But if I have to another change in this file I'll do it. Cheers